src/schema/v2/order/index.ts

```diff
@@ -1,169 +1,3 @@
-import {
-  GraphQLEnumType,
-  GraphQLFieldConfig,
-  GraphQLList,
-  GraphQLNonNull,
-  GraphQLObjectType,
-  GraphQLString,
-} from "graphql"
-import { ResolverContext } from "types/graphql"
-import { InternalIDFields } from "../object_identification"
-import { LineItemType } from "./lineItem"
-import { FulfillmentOptionType } from "./fulfillmentOption"
-import { Money, resolveMinorAndCurrencyFieldsToMoney } from "../fields/money"
-
-const OrderModeEnum = new GraphQLEnumType({
-  name: "OrderModeEnum",
-  values: {
-    BUY: {
-      value: "BUY",
-    },
-    OFFER: {
-      value: "OFFER",
-    },
-  },
-})
-
-const OrderSourceEnum = new GraphQLEnumType({
-  name: "OrderSourceEnum",
-  values: {
-    ARTWORK_PAGE: {
-      value: "ARTWORK_PAGE",
-    },
-    INQUIRY: {
-      value: "INQUIRY",
-    },
-    PRIVATE_SALE: {
-      value: "PRIVATE_SALE",
-    },
-    PARTNER_OFFER: {
-      value: "PARTNER_OFFER",
-    },
-  },
-})
-
-export const OrderType = new GraphQLObjectType<any, ResolverContext>({
-  name: "Order",
-  description: "Buyer's representation of an order",
```

Comment:
Most of this file moved into `sharedTypes.ts` or `MeOrder.ts` for the existing `me.order` query field